### ISSUE TITLE
github action check status is "neutral" when tests pass

### ISSUE THREAD

**Issue by gotchipete** [game-ci/unity-test-runner#220]
**Bug description**

When all tests pass, github still has the action marked as "neutral" instead of a pass / green check mark 

**How to reproduce**

Run the test runner with a suite of passing tests, observe a neutral result (see screenshot): 
![image](https://github.com/game-ci/unity-test-runner/assets/96428543/4ccf1b74-612d-461b-aa0a-e3ed543e49d2)

- **Expected behavior**

Passing tests should show a green check mark for the github action

**Comment by davidmfinol** [comment]
Personally, I like to refer to the status of the job that runs the unit tests to see if tests have passed or not.
I also like that the unit test results show as neutral, so that they are easy to find and know that they are separate from the other jobs.

**Comment by timcassell** [comment]
You could use a different test reporter to show the green checkmark, like dorny/test-reporter. See comments in #142 for details.

**Comment by gotchipete** [comment]
> You could use a different test reporter to show the green checkmark, like dorny/test-reporter. See comments in #142 for details.

Ahh I was looking for one, thank u @timcassell !

**Comment by davidmfinol** [comment]
> Will the job actually report a fail or success in this case, since the step will still be reporting neutral? 

Yep, I look at the status of the test job to see if that job actually passed or failed. I wouldn't call the test report a "step". It's just additional information that supplements the actual step that is the test job.

And like @timcassell suggests you could also use a different test reporter, or you could fork this repo and have your fork modify https://github.com/game-ci/unity-test-runner/blob/31086d985910613d75c32ba965f657df9c298820/src/model/results-check.ts#L91 to change the conclusion from `neutral` to `success`.

**Comment by gotchipete** [comment]
Ok, thanks.  I'm looking for the step to report pass/fail b/c I simply want
to fail the build and disallow a merge if test fail.  Seems like a fairly
common use case to me 🤔 ... but I'll look at the reporter or forking.
Thanks for the work on everything else that makes this work!

**Comment by frostebite** [comment]
This is already labeled `wontfix` by a maintainer but was never actually closed — closing now to reflect that decision and keep the open-issue list meaningful. If the `wontfix` label was applied in error, please reopen with the reasoning for why this should be revisited.
